.github/workflows/update-webassets.yaml

```diff
@@ -0,0 +1,55 @@
+name: Update Webassets
+
+on:
+  push:
+    branches:
+      - master
+      - teleport-v**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Get webapps, and the webapps.e submodule
+      - name: Checkout webapps
+        uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          submodules: recursive
+          fetch-depth: 0
+
+      # Get webassets, and the webassets.e submodule
+      - name: Checkout webassets
+        uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          repository: gravitational/webassets
+          submodules: recursive
+          path: dist
+
+      # This workflow only runs on push to a protected branch, so the code
+      # has already been reviewed and is trusted. This means it is safe to
+      # the version of the action that we checked out and we don't need to
+      # always use master.
+      - name: Build webassets
```

Comment:
Could we use https://github.com/docker/build-push-action instead? It supports build args, and allow us to use GHA caching for speedier CI runs